src/ComponentGallery/ComponentGallery.Server/ComponentGallery.Server.csproj

```diff
@@ -5,18 +5,27 @@
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
     <SpaRoot>..\ComponentGallery.Client</SpaRoot>
+    <SpaProxyLaunchCommand>npm run dev</SpaProxyLaunchCommand>
```

Comment:
mi troche to przeszkadzało to usunąłem, rozumiem, że to po to żeby errora w konsoli nie było?